src/generic/datavgen.cpp

```diff
@@ -864,7 +864,9 @@ class wxDataViewMainWindow: public wxWindow
     bool EnableDragSource( const wxDataFormat &format );
     bool EnableDropTarget( const wxDataFormat &format );
 
-    void RemoveDropHint();
+    void RemoveDropHint(bool refreshOnly = false);
```

Comment:
Please don't use boolean parameters, seeing `RemoveDropHint(true)` doesn't tell you anything about what this function does (differently from its call without `true`).

In this particular case, I don't even see why do we need to have this parameter at all, there should be just a separate function `RefreshDropHint()` and this one would simply call it. Please change it like this to make things simpler.